packaging/rpm/check_rpm_contents.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+if [ "$#" -ne 2 ]; then
+    echo "Usage: $0 <RPM_FILE> <EXPECTED_LIST_FILE>"
+    exit 1
+fi
+
+RPM_FILE="$1"
+EXPECTED_LIST_FILE="$2"
+
+if [ ! -f "$RPM_FILE" ]; then
+    echo "Error: RPM file not found at '$RPM_FILE'"
+    exit 1
+fi
+
+if [ ! -f "$EXPECTED_LIST_FILE" ]; then
+    echo "Error: Expected list file not found at '$EXPECTED_LIST_FILE'"
+    exit 1
+fi
+
+ACTUAL_SORTED=$(mktemp)
+EXPECTED_SORTED=$(mktemp)
+
+trap 'rm -f "$ACTUAL_SORTED" "$EXPECTED_SORTED"' EXIT
+
+echo "--- Analyzing RPM package: $RPM_FILE ---"
+
+rpm -qlp "$RPM_FILE" | sort > "$ACTUAL_SORTED"
+sort "$EXPECTED_LIST_FILE" > "$EXPECTED_SORTED"
+
+DIFFERENCES=$(diff "$ACTUAL_SORTED" "$EXPECTED_SORTED")
```

Comment:
can this line be deleted?

Reply:
i guess so 👍 https://github.com/apache/nifi-minifi-cpp/pull/1993/commits/786fa3aebef8f854f9429092f04a5c3806498e71#diff-1f580d23e862f5a2463f0da311830fc975ef8457bac86d92620693cc8f5d3988L31-L32